exchange/client.wasm.go

```diff
@@ -0,0 +1,27 @@
+//go:build wasm
```

Comment:
In some comments like this you have:

1. Space after `//`
2. `+` symbol

If possible making spacing consistent.

Also explain the `+` symbol with context in the MR description so that reviewers get what they are looking at

Reply:
`+`  symbol is from github ui shows it is added

Reply:
Look at other files. You have `// +blah`

Reply:
i think syntax is wrong for // +build !wasm will verify that :thinking: